SocialCareCaseViewerApi/V1/Domain/CaseStatus.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using SocialCareCaseViewerApi.V1.Infrastructure;
+
+#nullable enable
+namespace SocialCareCaseViewerApi.V1.Domain
+{
+    public class CaseStatus
+    {
+        public long? Id { get; set; } = null!;
+        public string? Type { get; set; } = null!;
+        public string? SubType { get; set; } = null!;
+        public string? StartDate { get; set; } = null!;
```

Comment:
no need for the `null!`, that's only required when a value cannot be null but is not initalised by the class.

Or alternatively if those values will never be null instead of saying `long?` just say `long`, the ? means it is nullable.